README.md

```
@@ -1,29 +1,28 @@
 ---
 description: >-
-  KIIEX is a hybrid centralized-decentralized exchange platform designed for
-  on-chain FX trading, cross-border payments and finance solutions.
+  KIIEX is a hybrid centralized-decentralized exchange platform designed for on-chain FX trading, cross-border payments, and finance solutions.
 ---
 
 # What is KIIEX?
 
-### What is KIIEX
+### What is KIIEX?
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)